src/Sulu/Bundle/AdminBundle/Resources/js/components/QRCode/tests/__snapshots__/QRCode.test.js.snap

```diff
@@ -13,2214 +13,20 @@ Array [
   <svg
     class="qrcode"
     height="256"
-    viewBox="0 0 256 256"
```

Comment:
Why this looks unexpected it still works like it should. We only added the viewBox as it did not work without it. Seems like they fixed the bug. For backward compatibility we still le our definition of `viewBox="0 0 256 256"` in it.